decidim-core/app/views/decidim/errors/internal_server_error.html.erb

```diff
@@ -1,28 +1,34 @@
-<div class="wrapper">
-  <div class="row">
-    <div class="large-8 large-centered">
-      <div class="static__message">
-        <h1 class="heading1 page-title"><%= t(".title") %></h1>
-        <p><%= t(".try_later") %></p>
-        <div class="table-scroll">
-          <table class="table-list">
-          <% @info_hash.map do |key, val| %>
-            <tr>
-              <th><%= t(".#{key}") %></th>
-              <td><%= val %></td>
-            </tr>
-          <% end %>
-          </table>
-        </div>
-        <textarea id="referenceData" class="hide"><%= @plain_info %></textarea>
-        <button class="button primary float-right"
-          data-clipboard-copy="#referenceData"
-          data-clipboard-copy-label="<%= t(".copied") %>"
-          aria-label="<%= t(".copy_error_message_clarification") %>">
-            <%= icon "clipboard", role: "img", "aria-hidden": true %>
-            <%= t(".copy_to_clipboard") %>
-        </button>
-      </div>
+<%= render layout: "layouts/decidim/shared/layout_center" do %>
+
+  <div class="place-self-center">
+    <div class="flex justify-center">
+      <h1 class="h3 md:h1 decorator my-12"><%= t("title", scope: "decidim.errors.internal_server_error") %></h1>
+    </div>
+
+    <div class="text-center space-y-8">
+      <p class="text-lg text-gray-2">
+        <%= t("try_later", scope: "decidim.errors.internal_server_error") %>
+      </p>
+
+      <table class="w-full border-separate border-spacing-2">
+        <% @info_hash.map do |key, val| %>
+          <tr>
+            <th class="text-right w-1/2"><%= t(key, scope: "decidim.errors.internal_server_error") %></th>
+            <td class="text-left w-1/2"><%= val %></td>
+          </tr>
+        <% end %>
+      </table>
+
+      <textarea id="referenceData" class="reset-defaults" hidden><%= @plain_info %></textarea>
+
+      <button class="button button__lg button__secondary"
```

Comment:
I may have a problem with this particular piece of code ... 
In my opinion, if there  is an asset issue (missing js file), the **Copy to clipboard** may not work as expected (or may not work at all)

Maybe it would be ok to have a local snippet that would handle the copy to clipboard functionality ?